### ISSUE TITLE
The plugin is not available on VSCode Insiders V1.32

### ISSUE THREAD

**Issue by Dodotry** [shd101wyy/markdown-preview-enhanced#1074]
Extension 'shd101wyy.markdown-preview-enhanced' uses the 'vscode.previewHtml' command which is deprecated and will be removed soon. Please file an issue against this extension to update to use VS Code's webview API.

Right-click to export HTML or open the menu in chrome and nothing happens.

**Comment by hqzxjczx** [comment]
Extension 'shd101wyy.markdown-preview-enhanced' uses the 'vscode.previewHtml' command which is deprecated and will be removed soon. Please file an issue against this extension to update to use VS Code's webview API.

Right-click to export HTML or open the menu in chrome and nothing happens. @shd101wyy

**Comment by shd101wyy** [comment]
Yes I will issue a new release by the end of this week. Thank you!
